package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "facnrecipev4",
+  "version": "4.0.0",
+  "description": "The ultimate FACN Recipe app",
+  "main": "index.js",
+  "scripts": {
+    "test": "nyc istanbul cover tape /tests/router.test.js | tap-spec",
+    "start": "node src/index.js",
+    "devStart": "nodemon src/index.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/facn2/facnrecipev4.git"
+  },
+  "author": "Leo and Stefano",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/facn2/facnrecipev4/issues"
+  },
+  "homepage": "https://github.com/facn2/facnrecipev4#readme",
+  "dependencies": {
```

Comment:
the ones that are there should be the ones that we are going to use, we plan ahead.